index.html

```diff
@@ -0,0 +1,108 @@
+</head>
+<body>
+    <div class="navbar">
+        <a href="https://www.youtube.com/">
+            <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAR8AAACwCAMAAAABtJrwAAAAwFBMVEX////+AAAoKCgAAAAkJCQaGhoXFxeBgYHc3NyampoTExMcHBwPDw87Ozuqqqq5ubnDw8NOTk5DQ0OJiYn4+Pj/5eUICAjv7+/o6OiioqLR0dHAwMArKyvg4OD/+vpiYmJ8fHz/8vJzc3P+xsb+JCT+OTn/2tqSkpJdXV3+pqb+hob+LCz+dHT/urr/0tL+RET+YGD/mpo0NDT+rq7+vL3+gYL+WVn+UVH+GxtISEj+aWn/1tb+PDz/6ur+j49qamqMHMNJAAAG2UlEQVR4nO2baYOaOhSGEeIGuI2I+4zLjDpLp+Mybaf3tv7/f3XJSUBQWVzQae/7fClIPJhnQnKSUEUBAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAMCfyq3g2j/jczCcjFeP86f3l7dv3x+en+/v7zMuzvHz88P3b28v70/zxy/jyfB/5mzy9JA5jIf310v9uGbfodC81O12uf15oBzB9+Flft5U47Qvc7M9DO/jXeznlxuiVao53LinH/ys1I+9s0Vf83NT3i1VzKmqqu+5cCG+Hqsn88OLoRqMMa0gTvoa48Tf2RIFfZh7msmV/Xw5Wk8mM3eD5A1fHSqmc2IM4m9taeoW+ufzc2jPHMANUrV9SkiW2Y2/9Z/gZ3iKnsxERqGaMtkB3TDnREsw4FjU82qM1Jh0XNwtdV0/45P8LN0wNXJi8UPhqpbk5gVOf8GovdHJHqvX9fN4kp8nN0ybV8Lu8MMOf9ZyexpCGCVyGzrcXdfP+0l+3twwBd5ocm2vPu5YlgThJ/QL1/XzdpKfZy8O75ONNT+649VNMLp7fGo/ocPXj0SCvDgjRxDr8SNeW2N0wE/41H5Ca/4rUc/kxaGkR2vx7JD3thX6sDpaOGPSYlSVhbrFdrtdpLM2HZKUjZ8+fUhPaZUOeZLg+Sl/aJpat7xbdil6adS5ip+GcvsS78ebgzU1mfQIUXwcan5oBj1rhlYTtRpouVxOq5MUkx9Wgn66/LpJrbBORQc+PzWeCDC9JwU1b7zod+npuY3w48zrY+ceDS8Sr2V2KrJDyoQs1fDyvqxIjShzzJIfSpLMLT9V7paVyE+WP6Z5z09llhWhcnkK5Y+ur1Pz8yum5qsYPxMv0jQrsh5e8ZzjSRnk6LdrVC+zeJoftWTkTJFIiiwrzz9ltoiupfaINWJbxjzSz2YVqMMfMF2xTPl76YFT7Xa/rpOnE/0wrdilJFTMXCg6W3RaU/3Q4eDMfpRh1PLQeBNKdNCUCGWd07LuTsmoWnb1ND/URGx6wHjjpF6OTJUOTScOYhLvR1H+CZ/DrjaleN5jdiu6zIMoDTK5lbbu1up4P2LuO/CyLDqiUaDuHaVBeP/S8BdbhpX6sinDG0y2ODVkstJjcsAXk3t2d5Ifkf+QaZra0XdM3hO1D03X0/CjKL9j/fDEh414s+FWxNoF/V0Loq84gx96qigSDV70VNFzbFeVdAhfHdvyo+xPhnx+qMX0erKCLdE9W64fVTuDH2qJvHdr0kHPcybz0fMTvrwR9BPm0e8nLzMSElBIww8tDfBIwr76WfxM/g0r5euf5V9XLnOIqpzZjxdJ+LlA+0kyfkUN8H4/lvRD3eaun+b5/fT7/Rb1z6n5SZD/RCaIY38wkb/RSHURP6rtzE8pO7qen1X0OsfEH0xkcv6uImU/Hqn5GUb7acRNUAN+5M/vX8xPTpdoafmJnL8nWOAIjHJiyiXmjxfx03YpppUfRqyPJVq6D+zBR/tRzj6+99KSksTP8jmBnkwglt/PxfKflDl+9z3az07+bJ7BD13nzYZWUaSfVtNS0iPBGmoEX0P97My/SmfwQ3Mt9qH4518tzTaN9BbIQuadCfkZ6kdRvbGsS+8rrF0/VOnj/NCHlF/dePu1ZD89P6e8vuF7gWPXz0zsGrsVpA3VKS2Krj0pO37EApgSUOnzQyWz3koSrZ6I/YD4l42OJHyCkYRA+hz0Q2s11Gpof53+wvR8qAa/Sh3Ith/ZkztFO3R944f1HBcV0kfOu94mNv0d7PS6oJP8BF+xC/gRHbQ+qsz4n1q8LiXaB/uY1szsPj/uCy/Ttdi78a8/27OZ6JRFfOG3Xr4je7PU9CjHvXwoCHbPQT/KiOpl6DQpEwmurD/LMrtc2uNHzuDUrGGM+BPk87NgTFyTa/FF0x89xS3C8BlYPMHHa8uPtZA7Vrz9yx2qvGgCqj4jFTt+qrIBMcPiWjZ+7Jbc3RGdDlepu8GZNk1PT9wGThQvW5Gamu3g+lGsO03n7yUa5qYCNefBYVnN0ZV3CotW1eNfkyvIdX7d0G5aSoVfHyj8/VXnYKG0Ss4lQ980lQFFN7KakdbkS3LsEL+tR7GqxOaDfns0q80GZd/uQmVUm9V5Hfu8KLUF8S1ZppOv1Qa0w8U/LLgF+Te6+bt10ReqVR7MZutpWkvPG15DlwcjuF/FB/5raCyf3h6Svgd9//D2tJzEB/37uB02Jq/j1Wq1fHx8nM/nvznOv87Z0vl0/DppDPH/VQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAMCF+Q/kHZDX+0FN3QAAAABJRU5ErkJggg==" width="150" height="50" alt="Youtube Logo">
+        </a>
+        <div class="search-container" style="padding-left: 350px">
```

Comment:
Inline styles are generally frowned upon, please move all inline styles to the css file